modules/fileout_netcdf/FONcDim.cc

```diff
@@ -101,6 +106,30 @@ void FONcDim::define(int ncid)
     }
 }
 
+void FONcDim::define_struct(int ncid)
+{
+    if (!_defined) {
+        if (_name.empty()) {
+            ostringstream dimname_strm;
```

Comment:
The stringstream classes are pretty slow. Another way to do this (with C++-14) is: _name = "sdim" + to_string(++FONcDim::StructDimNameNum);

Reply:
Yes, the code is changed by following your suggestion.